api/client.go

```diff
@@ -1253,16 +1253,8 @@ START:
 	}
 
 	if outputPolicy {
-		// We don't want to use a wrapping function or any special flags
-		outputPolicyClient, err := c.Clone()
-		if err != nil {
-			return nil, fmt.Errorf("unable to clone client for -output-policy: %v", err)
-		}
-		outputPolicyClient.config.OutputCurlString = false
-		outputPolicyClient.config.OutputPolicy = false
 		LastOutputPolicyError = &OutputPolicyError{
-			Request:     req,
-			VaultClient: outputPolicyClient,
+			Request: req,
```

Comment:
This is a little confusing since I think `Request` is actually part of the `*retryablehttp.Request` that you're embedding. Since you've done all this cleanup I'd suggest just taking in the method and path during the creation of the `OutputPolicyError`